tests/pos/i3539c.scala

```diff
@@ -0,0 +1,3 @@
+class Test {
+  asInstanceOf.asInstanceOf[AnyRef => String].apply(null)
```

Comment:
There is a typo in the original issue. I meant to write:
```scala
asInstanceOf[Nothing].asInstanceOf[AnyRef => String].apply(null)
```

Reply:
Changed and squashed in one file